validator/src/main.rs

```diff
@@ -392,6 +392,18 @@ fn hash_validator(hash: String) -> Result<(), String> {
         .map_err(|e| format!("{:?}", e))
 }
 
+fn default_fifo_shred_storage_size(vc: &ValidatorConfig) -> u64 {
+    // This is an estimation that is slightly larger than average so that we
+    // can make sure FIFO rarely evits data/coding shreds that is younger
+    // than the configured max_ledger_shreds while avoiding allocating too
+    // much disk space for data/coding shreds.
+    const AVG_SHRED_SIZE_BYTE: u64 = 3200;
```

Comment:
This number assumes 1:1 data:coding, right? A little more detail on where this number comes from would probably be good

Reply:
This is the total shred size, including both data and coding shred.

Reply:
I meant to include more detail on where you got 3200. Ie, shreds are 1228 bytes, we have 1:1 data and coding so double that, and add some margin for meta-data to get to 3200.

Or did you just double this number from ledger_cleanup_service comments

Reply:
I think the confusing part is the `/2` in the final computation.  

    Some(max_ledger_shreds) => (max_ledger_shreds / 2) * AVG_SHRED_SIZE_BYTE
    
I will change it to 1600 instead and remove the `/2` and add more comments.